Factory/HttpFoundationFactory.php

```diff
@@ -141,7 +141,7 @@ protected function getTemporaryPath()
     public function createResponse(ResponseInterface $psrResponse)
     {
         $cookies = $psrResponse->getHeader('Set-Cookie');
-        $psrResponse = $psrResponse->withHeader('Set-Cookie', []);
```

Comment:
I think this should be backported to 1.1.1 because it doesn't work with `zendframework/zend-diactoros:1.8.6` which requires PHP 7.0

Reply:
See https://www.drupal.org/project/drupal/issues/3039611#comment-13037090 for more details.